plugins/lime-plugin-fbw/src/actions.js

```diff
@@ -0,0 +1,28 @@
+import {
+	FBW_SEARCH_NETWORKS,
+	FBW_SET_NETWORK,
+	FBW_CREATE_NETWORK
+} from './constants';
+
+export const searchNetworks = ( rescan ) => ( dispatch ) => {
+	dispatch({
+		type: FBW_SEARCH_NETWORKS,
+		payload: {
+			rescan
+		}
+	});
+};
+
+export const setNetwork = ( file ) => ( dispatch ) => {
+	dispatch({
+		type: FBW_SET_NETWORK,
+		payload: file
+	});
+};
+
+export const createNetwork = (network) => (dispatch) => {
```

Comment:
Is there a reason why some parentheses have spaces inside like ```( file )``` but here ```(network)``` is without?

Reply:
Not to my knowledge.

Reply:
Okay, thanks for your input. That's where I applied the changes. It's rare to see someone here looking at the code. Are you a user of libremesh? Are you interested in contributing or coordinating some revisions?

Reply:
I was just browsing through and noticed that there are plenty of inconsistencies in the code style.